docs/manual/cli/user.en.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "contao:user"
+description: "Create and list Contao users."
+aliases:
+    - /en/cli/user/
+---
+
+
+After the [database](/en/cli/migrate/) has been created, Contao [users](/en/user-management/users/) can be created here. You can get 
+an overview of the commands via:
+
+```bash
+php vendor/bin/contao-console contao:user --help
+```
+
+## contao:user:list
+
+Lists existing Contao users:
+
+| Option | Beschreibung |
+| --- | --- |
+| `--help`   | Options overview |
+| `--admins` | Show administrators only |
```

Comment:
```suggestion
| `--column=COLUMN` | The columns display in the table (multiple values are allowed) |
| `--admins` | Show administrators only |
| `--format=FORMAT` | Output format (txt, json) |
```

There is two more options that could be added.
I guess this example could kinda explain it:
```cli
contao:user:list --column=name --column=admin --format=json
```
returns
```json
[{"name":"Contao","admin":1},{"name":"Demo","admin":0}]
```